repondeur/zam_repondeur/fetch/senat/derouleur.py

```diff
@@ -127,22 +127,23 @@ def _parse_derouleur_data(data_iter: Iterable[Any]) -> List[DiscussionDetails]:
 
 
 def derouleur_urls(lecture: Lecture) -> Iterator[str]:
+    assert lecture.texte.session_str is not None  # mypy hint
```

Comment:
I'm surprised this line doesn't generate an error on the bandit side.

Reply:
Yup

Reply:
Rebase manually, solved the conflicts, and bandit was indeed unhappy with the asserts.